examples/java/src/main/dafny/Foo.dfy

```diff
@@ -0,0 +1,5 @@
+module Foo {
+  datatype Bar = create(baz: string) {
```

Comment:
```suggestion
  datatype Bar = Create(baz: string) {
```

nit: [Dafny style guide ](https://dafny.org/latest/StyleGuide/Style-Guide#naming-convention) says to use PascalCase for Datatype